### DISCUSSION TITLE
Upgrade - Docker - Requirements

### DISCUSSION THREAD

**Discussion by pjwarner** [firefly-iii/firefly-iii#9594] [Firefly III]
I am using Portainer to manage my Firefly-III instance and I was curious if there was ever a need to upgrade the other modules when upgrading Firefly-III:Core?

For instance do I need to upgrade the Importer or CRON Container(s) or anything like that as well?  These are the only containers that I currently have set to latest but they dont upgrade unless i initiate the recreation of the docker image.  If there isn't a reason to upgrade those now (I am going from 6.1.19 to either 6.2.25 or alpha 1...whatever latest is in Dockerhub) where would I go to identify if a new version was being leveraged?

Thanks in advance!

**Comment by JC5** [comment]
Hey, thanks for starting a discussion. 

As a rule, you can update them when you update Firefly III. They are pretty stable and don't need many updates.
